spyder_unittesting/unittesting.py

```diff
@@ -37,7 +38,7 @@ def get_plugin_title(self):
 
     def get_plugin_icon(self):
         """Return widget icon"""
-        return get_icon('profiler.png')
+        return ima.icon('profiler')
```

Comment:
This is a leftover, this plugin has no icon yet. It's not really in a finished state...